providers/implementations/serializers/serializer_dsa_pub.c

```diff
@@ -106,15 +116,20 @@ static int dsa_pub_pem(void *ctx, void *dsa, BIO *out,
 static int dsa_pub_print_data(void *ctx, const OSSL_PARAM params[], BIO *out,
```

Comment:
If these functions remain almost identical then we may want to collapse them at some time in the future.

Reply:
Yeah, I was thinking exactly that. It's for another refactoring, another PR